src/components/create-application-form/index.js

```diff
@@ -85,126 +87,109 @@ export class CreateApplicationForm extends Component {
             label="Name"
-            help="Lower case; no spaces or special characters"
-          >
-            <Input
-              type="text"
-              variant="default"
-              name="name"
-              value={this.state.form.name}
-              onChange={this.handleNameChange}
-            />
-          </FormField>
-          <FormField
+            helperText="Lower case; no spaces or special characters"
+            name="name"
+            value={this.state.form.name}
+            onChange={this.handleNameChange}
+          />
+          <TextField
             label="GitHub repository"
-            help="Full URL, e.g. 'https://github.com/equinor/my-app'"
-          >
-            <Input
-              type="text"
-              variant="default"
-              name="repository"
-              value={this.state.form.repository}
-              onChange={this.makeOnChangeHandler()}
-            />
-          </FormField>
-          <FormField
+            helperText="Full URL, e.g. 'https://github.com/equinor/my-app'"
+            name="repository"
+            value={this.state.form.repository}
+            onChange={this.makeOnChangeHandler()}
+          />
+          <TextField
             label="Config Branch"
-            help="The name of the branch where Radix will read the radixconfig.yaml from, e.g. 'main' or 'master'"
-          >
-            <Input
-              type="text"
-              variant="default"
-              name="configBranch"
-              value={this.state.form.configBranch}
-              onChange={this.makeOnChangeHandler()}
-            />
-          </FormField>
-          <FormField
+            helperText="The name of the branch where Radix will read the radixconfig.yaml from, e.g. 'main' or 'master'"
+            name="configBranch"
+            value={this.state.form.configBranch}
+            onChange={this.makeOnChangeHandler()}
+          />
+          <TextField
             label="Owner"
-            help="Owner of the application (email). Can be a single person or shared group email"
-          >
-            <Input
-              type="email"
-              variant="default"
-              name="owner"
-              value={this.state.form.owner}
-              onChange={this.makeOnChangeHandler()}
-            />
-          </FormField>
+            helperText="Owner of the application (email). Can be a single person or shared group email"
+            name="owner"
+            value={this.state.form.owner}
+            onChange={this.makeOnChangeHandler()}
+          />
           <AppConfigAdGroups
             adGroups={this.state.form.adGroups}
             adModeAuto={this.state.form.adModeAuto}
             handleAdGroupsChange={this.makeOnChangeHandler()}
             handleAdModeChange={this.handleAdModeChange}
           />
-          <FormField
+          <TextField
             label="WBS"
             help="WBS of the application for cost allocation"
```

Comment:
field should be `helperText`
```suggestion
            helperText="WBS of the application for cost allocation"
```

Reply:
fixed